WeakAuras/Init.lua

```diff
@@ -20,6 +20,8 @@ WeakAuras.versionString = versionStringFromToc
 WeakAuras.buildTime = buildTime
 WeakAuras.printPrefix = "|cff9900ffWeakAuras:|r "
 WeakAuras.newFeatureString = "|TInterface\\OptionsFrame\\UI-OptionsFrame-NewFeatureIcon:0|t"
+WeakAuras.BuildInfo = select(4, GetBuildInfo())
+WeakAuras.IsClassic = WeakAuras.BuildInfo < 20000
```

Comment:
I would prefer something along the lines of

```lua
local isClassic = false
--[[@classic
isClassic = true
--@end-classic]]

function WeakAuras.IsClassic()
  return isclassic
end
```

At the very least, I would prefer that IsClassic is a function, similar to other status indicators like WeakAuras.InternalVersion() and WeakAuras.IsImporting(). Exactly how that information is gathered is something that I'm willing to let someone else decide about.